### ISSUE TITLE
[23w29d] Empty Gas Clouds Near Planets

### ISSUE THREAD

**Issue by DrizztCode** [tsunamayo/Starship-EVO#5181]
### Short Description of the issue:
Getting gas clouds in the targetting system with 0L of fuel and no visuals (other than a target outline), near the new planets
...

### Steps to reproduce the issue:
find a new planet, if it has gas clouds, they will likely be these empty ones
...

### Environment
- Starship EVO version: 23w29d
- Graphics Card: RTX 2060

### Additional information:

![20230721164205_1](https://github.com/tsunamayo/Starship-EVO/assets/104569043/1951f5fc-57f0-45a5-a9e8-0dab85a530d5)

**Comment by Crimson-Artist** [comment]
this has been a bug for awhile. it happens if you pass close enough to a star where the gas clouds start to load in. Once you leave the area the game still thinks that you're near the clouds so they keep rendering them so you can target it.

**Comment by DrizztCode** [comment]
> this has been a bug for awhile. it happens if you pass close enough to a star where the gas clouds start to load in. Once you leave the area the game still thinks that you're near the clouds so they keep rendering them so you can target it.

i don't think so - unless i misunderstood you

this is not near a star

these are other targets around the planet

e.g. the one in the screenshot is 36km from me - and you can see the planet in the background

i have never seen phantom gas clouds before in the game

i have seen them be targetted from a very long distance away - but they are the real clouds that have more than 0L of fuel in them

this is just an empty bunch of targets that are presenting as gas clouds in the targetting screen

i'm guessing it's a side effect of the new planets

are you saying that in previous versions of the game if you flew near the star and then flew away it would tell you there were empty gas clouds near you in other places?

**Comment by Crimson-Artist** [comment]
> 

yes this has been happening for quite some time. Im not sure why no one reported it before. it goes away when you log out of the game so its technically a low priority bug but a bug nonetheless

**Comment by DrizztCode** [comment]
> > 
> 
> yes this has been happening for quite some time. Im not sure why no one reported it before. it goes away when you log out of the game so its technically a low priority bug but a bug nonetheless

ah ok - that's interesting - and good to know

would also be somewhat fixed if my suggestion about being able to disengage the targeting interface and hud gets implemented :-)

thanks for the info :-)

and weird that i never noticed the empty clouds before

was possibly because i stopped going near the star every time i discovered that the fuel scoop wasn't quite there yet

anyway - definitely the reload is good to know - since if i'm near a planet after having been near the star, i'm likely not heading back to the star again for at least a little while

anyway - yeah - thanks for the info :-)
